zips/zip-0312.rst

```diff
@@ -163,44 +159,68 @@ following modifications.
 Randomizer Generation
 '''''''''''''''''''''
 
-A new helper function is defined, which generates a randomizer. The
-`encode_signing_package` is defined as the byte serialization of the `msg`,
-`commitment_list` values as described in [#frost-serialization]_.
-Implementations MAY choose another encoding as long as all values (the message,
-and the identifier, binding nonce and hiding nonce for each participant) are
+Two new helper functions are defined, which generate a randomizer; the first is
+used by the Coordinator and the second is used by participants. The
+`encode_signing_commitments` function SHOULD return the byte serialization of
```

Comment:
This is somewhat confusing, given that `random_bytes` gets its own paragraph. Separate `encode_signing_commitments` into its own paragraph, and then maybe turn those two paragraphs into bullet points? or something that clarifies these are _references_ to functions defined elsewhere that are used by the two new helper functions, not the two new helper functions being defined. Maybe also import the references first, and then say "Two new helper functions are defined" immediately before the actual definitions.

Reply:
Agreed, I restructured this in c74bed50